testutil/common/tester.go

```diff
@@ -117,22 +123,42 @@ func (ts *Tester) Accounts(name string) []Account {
 	return accounts
 }
 
-func (ts *Tester) StakeAccount(acct Account, spec spectypes.Spec, amount int64, geoloc uint64, moniker string) error {
+func (ts *Tester) StakeAccount(acct Account, spec spectypes.Spec, amount int64) error {
```

Comment:
the word account can be misleading as, can we have only stakeprovider method?

Reply:
fixed 4d40a32c